examples/simple-filtering/simple_filtering.description

```diff
@@ -0,0 +1,7 @@
+//Simple filtering can be performed for the events which come in the stream.
```

Comment:
// Simple filtering can be performed for the events that come in the stream.
// In this example, events are related to the person being monitored and there 
// will be an alert triggered when there is a person whose age is less than or equal to 16.
// This example can be extended to make simple filtering based on some condition.
// Ballerina Streams is an experimental feature. Please use `--experimental` flag when 
// compiling Ballerina files that have streaming constructs.

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/12750.